From: lemoer <git@irrelefant.net>
Date: Mon, 19 Jul 2021 01:20:11 +0200
Subject: TEMPORARY: migrate interface name "vpn" to "wg_mesh"!

diff --git a/package/gluon-mesh-vpn-wireguard/files/lib/gluon/upgrade/390-migrate-mesh-vpn-wireguard b/package/gluon-mesh-vpn-wireguard/files/lib/gluon/upgrade/390-migrate-mesh-vpn-wireguard
new file mode 100755
index 0000000000000000000000000000000000000000..2bead80c8cfa7575a73e2af9f7437c413116e857
--- /dev/null
+++ b/package/gluon-mesh-vpn-wireguard/files/lib/gluon/upgrade/390-migrate-mesh-vpn-wireguard
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# rename vpn interface to wg_mesh
+if uci -q get network.vpn > /dev/null; then
+	uci rename network.vpn=wg_mesh
+	uci commit network
+fi
