From: Sebastian Schaper <openwrt@sebastianschaper.net>
Date: Mon, 16 Oct 2023 18:10:06 +0200
Subject: ath79-generic: add support for TP-Link EAP-225 Outdoor v3

diff --git a/docs/user/supported_devices.rst b/docs/user/supported_devices.rst
index e5733e33dc8fee79db034bc9f73ba33cca7b8623..a41743c65582fcdb1888561be6f1bd57254f745a 100644
--- a/docs/user/supported_devices.rst
+++ b/docs/user/supported_devices.rst
@@ -115,6 +115,7 @@ ath79-generic
   - CPE510 (v1.0, v1.1, v2.0, v3.0)
   - CPE710 (v1.0)
   - EAP225-Outdoor (v1)
+  - EAP225-Outdoor (v3)
   - TL-WDR3500 (v1)
   - TL-WDR3600 (v1)
   - TL-WDR4300 (v1)
diff --git a/package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua b/package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua
index ba4b5b489250b860b1235493a26ea6cc8c807932..82c780ff4379a5dcebd8e5cd5e7314d196eef081 100644
--- a/package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua
+++ b/package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua
@@ -37,6 +37,7 @@ function M.is_outdoor_device()
 		'tplink,cpe510-v3',
 		'tplink,cpe710-v1',
 		'tplink,eap225-outdoor-v1',
+		'tplink,eap225-outdoor-v3',
 		'tplink,wbs210-v1',
 		'tplink,wbs210-v2',
 		'tplink,wbs510-v1',
diff --git a/targets/ath79-generic b/targets/ath79-generic
index 4a4426916729f789edb849c5b432d5e9f2375903..398449046d229fdfdb22bbf10f832aa7f835f174 100644
--- a/targets/ath79-generic
+++ b/targets/ath79-generic
@@ -467,6 +467,9 @@ device('tp-link-cpe710-v1', 'tplink_cpe710-v1')
 device('tp-link-eap225-outdoor-v1', 'tplink_eap225-outdoor-v1', {
 	packages = ATH10K_PACKAGES_QCA9888,
 })
+device('tp-link-eap225-outdoor-v3', 'tplink_eap225-outdoor-v3', {
+	packages = ATH10K_PACKAGES_QCA9888,
+})
 
 device('tp-link-tl-wdr3500-v1', 'tplink_tl-wdr3500-v1')
 device('tp-link-tl-wdr3600-v1', 'tplink_tl-wdr3600-v1')
